client_libraries-DEPRECATED/python/src/consumersurveys_v2_discovery.json

```diff
@@ -135,150 +193,122 @@
   "Survey": {
    "id": "Survey",
    "type": "object",
-   "description": "Message defining the Survey collections for the REST API.",
```

Comment:
This file was generated: https://www.googleapis.com/discovery/v1/apis/consumersurveys/v2/rest